Lib/fontbakery/profiles/googlefonts.py

```diff
@@ -6181,6 +6182,32 @@ def com_google_fonts_check_colorfont_tables(ttFont):
     else:
         yield PASS, "Looks good!"
 
+@check(
+    id = "com.google.fonts/check/space_on_gid1_for_colrv0",
+    rationale = """
+        A shaping bug in Windows 10 inserts whichever glyph is on GID 1
+        on top of color glyphs, spaces, and also seemingly at random,
```

Comment:
I don't get what that "and also seemingly at random" clause actually means. Better to simply like so:
".. on top of other glyphs, colored or not. This only occurs for COLR version 0 fonts."